state-chain/pallets/cf-lp/src/lib.rs

```diff
@@ -117,6 +117,12 @@ pub mod pallet {
 			chain: ForeignChain,
 			address: ForeignChainAddress,
 		},
+		LiquidityDepositCredited {
```

Comment:
could we expose the information necessary to identify the channel of the deposit here?

Reply:
chatted in dms, not necessary after adding the account id of the depositer to DepositReceived